### DISCUSSION TITLE
node version required

### DISCUSSION THREAD

**Discussion by Benleie** [vue-macros/vue-macros#422] [Q&A]
Just what to know, why Node.js 16.14.0 or higher isrequired when i use Vue Macros.
![image](https://github.com/sxzz/vue-macros/assets/24362871/c0fc6828-316a-4f5e-8b30-da8fe559567a)

**Comment by sxzz** [comment]
Because Node 14 has been EOL (2023-04-30)
https://github.com/nodejs/Release#release-schedule
